#!/bin/sh -e
## 60-hplj1000.dpatch, by Luca Capello <luca@pca.it>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use the same firmware folder for all HP LJ printers

if [ $# -lt 1 ]; then
echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
      -patch) patch $patch_opts -p1 < $0;;
      -unpatch) patch $patch_opts -p1 -R < $0;;
      *)
      echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
      exit 1;;
esac

exit 0

@DPATCH@
--- foo2zjs-20090301dfsg.orig/hplj1000.ORG	2008-06-05 12:02:14.000000000 +0200
+++ foo2zjs-20090301dfsg.orig/hplj1000	2009-04-02 17:02:11.000000000 +0200
@@ -77,37 +77,31 @@
     MODEL=P1005; FWMODEL=$MODEL
     USB1=0x03f0	#Vendor
     USB2=0x3d17	#Model
-    FWDIR=/usr/share/foo2xqx/firmware
     ;;
 *P1006)
     MODEL=P1006; FWMODEL=$MODEL
     USB1=0x03f0	#Vendor
     USB2=0x3e17	#Model
-    FWDIR=/usr/share/foo2xqx/firmware
     ;;
 *P1007)
     MODEL=P1007; FWMODEL=P1005		# Alias
     USB1=0x03f0	#Vendor
     USB2=0x4817	#Model
-    FWDIR=/usr/share/foo2xqx/firmware
     ;;
 *P1008)
     MODEL=P1008; FWMODEL=P1006		# Alias
     USB1=0x03f0	#Vendor
     USB2=0x4917	#Model
-    FWDIR=/usr/share/foo2xqx/firmware
     ;;
 *P1505)
     MODEL=P1505; FWMODEL=$MODEL
     USB1=0x03f0	#Vendor
     USB2=0x3f17	#Model
-    FWDIR=/usr/share/foo2xqx/firmware
     ;;
 *P1505n)
     MODEL=P1505n; FWMODEL=$MODEL
     USB1=0x03f0	#Vendor
     USB2=0x4017	#Model
-    FWDIR=/usr/share/foo2xqx/firmware
     ;;
 *1000)
     MODEL=1000; FWMODEL=$MODEL
